tensorflow/python/client/device_lib.py

```diff
@@ -18,6 +18,7 @@
 # pylint: disable=invalid-import-order, g-bad-import-order, wildcard-import, unused-import, undefined-variable
 from tensorflow.python import pywrap_tensorflow
 from tensorflow.python.client import _pywrap_device_lib
+from tensorflow.python.platform import tf_logging as logging
```

Comment:
For this import, you also need to change the BUILD file to include the target there.

```
ERROR: Strict deps violations: //tensorflow/python/client:device_lib
  In tensorflow/python/client/device_lib.py, no direct deps found for imports:
    line 21: from tensorflow.python.platform import tf_logging as logging: Module "tensorflow.python.platform" not provided by a direct dep
```